docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<p>Should we directly be calling <code>_SuperstaqClient</code> since its not meant to be part of the user interface as it is labeled with an underscore?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>According to Victory --&gt; There is not good reason to have made this function private so we can rename it to a function without the underscore.</p><p> </p><p>I will make a new PR to address the change.</p>

Reply:
I see, okay cool

Reply:
<p>https://github.com/Infleqtion/client-superstaq/issues/600</p><p> </p><p>Will be addressed in this issue.</p>